### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by ananya8226**
I have made this website responsive and changed some fonts. 
I aim to add some more styling and labels by tomorrow.

### CODE REVIEW

**Comment by Raghwendra-Dey** [comment]
Hey @ananya8226 great work!! 
Just few points I would like to add:
* Could you center the `Go` button and other buttons of the timers.
* Could you please add the subscript below the timers, which shows the name of each block, like, `seconds`, `minutes` etc.
* Could you round the corners of the buttons, like it is now.
* Please post some screenshots of the site in phone. Maybe, you can try to deploy your fork, and look at it from the phone. I can't deploy this without merging this.
* Most important: Please include the navbar, the quotes strip, and the background image which is there in the current site.

Tl;dr: please look into the current version of the site, and add all the things currently present, and make them responsive.

**Comment by Raghwendra-Dey** [comment]
Hey @ananya8226 are you done with the proposed changes in your patch?
